authors/apps/authentication/models.py

```diff
@@ -44,7 +44,7 @@ def create_superuser(self, username, email, password):
 
       user = self.create_user(username, email, password)
       user.is_superuser = True
-      user.is_staff = True
+      user.is_staff = False
```

Comment:
Why did you change the staff status

Reply:
It is an error on my part thanks for correcting the changes.